icgcget/cli.py

```diff
@@ -83,17 +98,19 @@ def cli(ctx, config, logfile):
 
     if logfile is not None:
         logger_setup(logfile)
-    elif config_file.has_key('logfile'):
+    elif 'logfile' in config_file:
         logger_setup(config_file['logfile'])
     else:
         logger_setup(None)
 
 
 @cli.command()
-@click.argument('repo', type=click.Choice(REPOS))
-@click.argument('fileid', nargs=-1)
-@click.option('--manifest', '-m', default=False)
+@click.argument('fileids', nargs=-1, required=True)
+@click.option('--repos', '-r', type=click.Choice(REPOS), multiple=True, required=True)
+@click.option('--manifest', '-m', is_flag=True, default=False)
 @click.option('--output', type=click.Path(exists=False))
+@click.option('--portal-api')
```

Comment:
I don't think we need this as it will always be the same value and clutters the command line. As for the other, I don't think that is needed either. Both of the them should be supported in the config file. Yes I know, we said that we should be able to override _everything_, but realistically this is only ever going to be used by us.

So is it currently possible to have a config file only variables? If not, maybe click supports hidden options.